website/content/docs/security/acl/tokens/create/create-a-snapshot-agent-token.mdx

```diff
@@ -0,0 +1,172 @@
+---
+layout: docs
+page_title: Create tokens for snapshot agents
+description: >-
+  Learn how to create an ACL token for the Consul snapshot agent.
+---
+
+# Create a snapshot agent token
+
+This topic describes how to create a token for the Consul snapshot agent.
+
+<EnterpriseAlert />
+
+~> The [`agent`](/consul/commands/snapshot/agent) subcommand described here is
+available only in [Consul Enterprise](https://www.hashicorp.com/products/consul/)
+version 0.7.1 and later. All other [snapshot subcommands](/consul/commands/snapshot)
+are available in the open source version of Consul.
+
+## Introduction
+
+The `consul snapshot agent` command starts a process that takes snapshots of the state of the Consul
+servers and either saves them locally or pushes them to a remote storage service. Refer to [Consul Snapshot Agent](/consul/commands/snapshot/agent) for additional information.
+
+## Requirements
+
+Core ACL functionality is available in all versions of Consul.
+
```

Comment:
```suggestion
### `agent` command requirements

The [`agent`](/consul/commands/snapshot/agent) subcommand requires [Consul Enterprise](https://www.hashicorp.com/products/consul/). All other [snapshot subcommands](/consul/commands/snapshot)
are available in the open source version of Consul.
```
The requirement is that the `agent` subcommand requires CE. We can lean on the version switcher, release notes, and changelog to communicate when the functionality changed.

If we place this here, we'll need a separate heading for the next section, maybe "Snapshot agent ACL requirements", so readers don't  think it all pertains to the agent subcommand.

Reply:
I added the `Snapshot agent ACL requirements` heading (hopefully in the right place)